### ISSUE TITLE
docs: citation

### ISSUE THREAD

**Issue by casperdcl** [iterative/dvc#3379]
- continues #3373
- https://github.com/iterative/dvc/blob/5101bf12531a2ef1782e47cb8cbd73d6843652c4/.zenodo.json#L6-L7
  + Not sure if we want `name: "DVC team"` or `name: "Iterative"` or `name: "iterative.ai"` or something else... Same with `"affilitation"`.
  + `affiliation` can be blank and will show up when `name` is hovered over.
  + `name` should really match whatever we choose for the citation text (i.e. think of it as an "author" field)
- https://github.com/iterative/dvc/blob/5101bf12531a2ef1782e47cb8cbd73d6843652c4/README.rst#L239-L240
  + perhaps drop "Git for Data & Models?"

**Comment by andronovhopf** [comment]
Yep I'd definitely drop "Git for Data & Models" in line 240 of README.rst
